scripts/pylint_extensions.py

```diff
@@ -1228,40 +1228,50 @@ def process_module(self, node):
             if line_num > 0:
                 previous_line = file_content[line_num - 1].strip()
 
+            # Comments may include a lowercase character at beginning
```

Comment:
Instead of special-casing like this, why not just include "coding: utf-8" as one of the allowed prefixes?

Reply:
If I do that then it will raise 'missing punctuation' warning.

Reply:
That's probably worth explaining in the code somewhere. Probably in the definition of the EXCLUDED_PHRASES variable.

Speaking of which, if that variable is only used in this method, then let's bring it into this method.

Reply:
EXCLUDED_PHRASES is also used in Docstring checker, so I think it is fine to keep it global.